impl/pom.xml

```diff
@@ -86,6 +86,11 @@
             <artifactId>javax.annotation-api</artifactId>
             <optional>true</optional>
         </dependency>
+        <dependency>
+            <groupId>org.apache.commons</groupId>
+            <artifactId>commons-lang3</artifactId>
+            <version>3.11</version>
+        </dependency>
```

Comment:
nit: don't include commons-lang for a single method, for `isNumeric` either use a regex `\\d+` (or something like that)